cmake/OpenCVDetectOpenCL.cmake

```diff
@@ -36,7 +36,7 @@ if(OPENCL_FOUND)
     set(HAVE_OPENCL_SVM 1)
   endif()
 
-  set(OPENCL_INCLUDE_DIRS ${OPENCL_INCLUDE_DIR})
+  set(OPENCL_INCLUDE_DIRS ${OPENCL_INCLUDE_DIR} CACHE PATH "OpenCL include directory")
```

Comment:
OPENCL_INCLUDE_DIRS should not be cached. Only `OPENCL_INCLUDE_DIR` is cached.

Reply:
OK